openreview/arr/process/proposed_assignment_pre_process.js

```diff
@@ -4,6 +4,7 @@ async function process(client, edge, invitation) {
   const committeeName = invitation.content.committee_name?.value;
   const { groups } = await client.getGroups({ id: invitation.domain });
   const domain = groups[0];
+  const quota = domain.content.reviewer_assignment_quota?.value
```

Comment:
what happens if this value is not being set?

Reply:
yes, I think we should check if there is a quota available then do the validation.